docs/ble.md

```diff
@@ -58,36 +73,36 @@ dictionary CharacteristicInit {
     NotifyCallback onNotify;            // optional
 };
 
-callback ReadCallback = void (unsigned long offset, FulfillReadCallback);
+interface Characteristic {
+    attribute ReadCallback onReadRequest;
+    attribute WriteCallback onWriteRequest;
+    attribute SubscribeCallback onSubscribe;
+    attribute UnsubscribeCallback onUnsubscribe;
+    attribute NotifyCallback onNotify;
+	attribute CharacteristicResult response;
+};
+
+callback ReadCallback = void (unsigned long offset,
+                              FulfillReadCallback fulfillreadcallback);
 callback WriteCallback = void (Buffer data, unsigned long offset,
-                               boolean withoutResponse, FulfillWriteCallback);
+                               boolean withoutResponse,
+							   FulfillWriteCallback fulfillwritecallback);
```

Comment:
really weird indentation

Reply:
Yeah, sorry; if you ever see this again, it's b/c Emacs tabs have to be converted to spaces for the Markdown to line up correctly.